src/App/Utilities/ThemeManager.cs

```diff
@@ -14,61 +17,79 @@ public static class ThemeManager
 
         public static void SetThemeStyle(string name, ResourceDictionary resources)
         {
-            Resources = () => resources;
+            try
```

Comment:
NP, code style suggestion:

```csharp
        private static (ResourceDictionary Resources, bool UseLightTheme) GetConfig(string name) => name switch
        {
            "dark" => (new Dark(), false),
            "black" => (new Black(), false),
            "nord" => (new Nord(), false),
            "light" => (new Light(), true),
            _ => OsDarkModeEnabled() ? (new Dark(), false) : (new Light(), true)
        };

//..

            var config = GetConfig(name);
            resources.MergedDictionaries.Add(config.Resources);
            UsingLightTheme = config.UseLightTheme;

```

Reply:
@vvolkgang  The other day I tested the new c# features but weren't compiling because the project is set with target framework Net Standard 2.0 thus it accepts C# 7.3 by default (and it's marked as using Default c# lang) [https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults).
We might need to change that config in order to use new stuff and also check whether the CI configuration is also able to compile on latest C#

Reply:
Fair point, we'll merge this for now and deal with that later! Thanks!